UnityProject/Assets/Scripts/Systems/Inventory/Pickupable.cs

```diff
@@ -301,6 +325,17 @@ private void CheckSpriteOrder()
 	public void _SetItemSlot(ItemSlot toSlot)
 	{
 		this.itemSlot = toSlot;
+		if (isServer)
+		{
+			if (toSlot != null && toSlot.ItemStorage != null)
+			{
+				clientSynchronisedStorageIn = toSlot.ItemStorage.gameObject.NetId();
+			}
+			else
+			{
+				clientSynchronisedStorageIn = NetId.Empty;
+			}
+		}
```

Comment:
```cs
if (isServer == false) return;
if (toSlot == null || toSlot.ItemStorage == null) 
{
   clientSynchronisedStorageIn = NetId.Empty;
   return;
}
clientSynchronisedStorageIn = toSlot.ItemStorage.gameObject.NetId();
```

Reply:
I feel like the if statement you posted is more complicated than the original

Reply:
I'll have a crack at this, 😛 

```csharp
if (isServer)
{
    clientSynchronisedStorageIn = toSlot?.ItemStorage.OrNull()?.gameObject.NetId() ?? NetId.Empty;
}
```

(not a review)